segments/path_read_only.bash

```diff
@@ -0,0 +1,17 @@
+### Defaults
+_sbp_path_color_readonly_fg=${_sbp_path_color_readonly_fg:-$_sbp_color_white}
+_sbp_path_color_readonly_bg=${_sbp_path_color_readonly_bg:-$_sbp_color_red}
+
+function _sbp_generate_path_read_only_segment {
+  if [ ! -w $PWD ] ; then
```

Comment:
I think we should go with double brackets and also quote the variable for good measure. I've tested it 'as is' in a folder with spaces in the name and it still worked fine, but I think it's good practise to be consistent on this.